buildspec.yml

```diff
@@ -45,7 +45,7 @@ phases:
       - echo Build started on `date`
       - echo Building Node app...
       - npm run build
-      - CI=true npm test
+      - CI=true npm run test:cypress
```

Comment:
endre dette til å også stemme overens med husky pre-commit.

Reply:
husky pre-commit er tatt med nå. men den kjører kun prettier, ikke test.

Reply:
Til meg så kjører den husky. men man må installere husky lokalt også for at den skal begynne å virke. Har du kjørt `npx husky install`?

Reply:
Spørsmålet er også om vi vil at den skal kjøre alle cypress testene hver bidige gang man commiter. La til npm test på det tidspunktet vi ikke fikk tilbakemeldinger fra aws hvordan bygginga gikk, slik at man i det minste kunne være sikker på at ting kjørte lokalt før man klønet det til å merget til master uten å sjekke hvordan det sto til med aws.